lib/Basic/CmdLineOptions.cpp

```diff
@@ -117,6 +117,12 @@ llvm::cl::opt<bool>
     UseAssignmentValidatingSolver("debug-assignment-validating-solver",
                                   llvm::cl::init(false));
 
+llvm::cl::opt<bool>
+ExecuteFloatAsInt("execute-float-as-int",
+                  llvm::cl::desc("Switch on numerical precision analysis "
```

Comment:
@Himeshi Perhaps it is clearer to say `Scale numerator of divisions to prevent rounding the result to zero`.